plugins/org.jboss.reddeer.eclipse/src/org/jboss/reddeer/eclipse/debug/ui/launchConfigurations/LaunchConfigurationsDialog.java

```diff
@@ -206,6 +206,12 @@ public String errorMessageWhile() {
 		public String errorMessageUntil() {
 			return "Tree does not has focus or TreeItem '"+item.getText()+"' is not selected";
 		}
+
+		@Override
```

Comment:
Well question is whether AbstractWaitCondition is proper place for default implementation of the method - if there are classes using WaitCondition instead of AbstractWaitCondiition, it would be better to put default impl to the interface.